types/name-initials/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for name-initials 0.1
+// Project: https://github.com/ericvera/name-initials#readme
+// Definitions by: Shirsh Zibbu <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/name-initials/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
fixed in f118ba0